web/css/doppelkopf.css

```diff
@@ -0,0 +1,26 @@
+.mainArea {
+  width: 1000px;
+  padding: 10px;
+  border: 1px solid black;
+}
+fieldset {
+  padding: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+.schreibTabelle{
+  border-collapse:collapse;
+	border:1px solid black;
```

Comment:
Tabs oder Spaces? In Zeile 12 sind 2 Spaces, in Zeile 13 ein Tab. Das zikeht sich auch durch die weitere Datei (z.B. 15-16 vs. 18-19). Am besten immer einheitlich Tabs und Spaces verwenden.

Außerdem ist manchmal nach dem Doppelpunkt ein Leerzeichen, manchmal nicht, vgl `padding: 10px;` vs. `border-collapse:collapse;`

Reply:
Ich habe bei dem Branch noch `stylelint` hinzugefügt. Das ist wie ESLint, nur eben für CSS. Das heißt, du solltest den Branch bei dir nochmal pullen, und danach `npm install` ausführen, damit du stylelint auch bei dir installierst.

Dann brauchst du für VSCode nur noch die stylelint Extension. Da gibt es mehrere, aber so wie ich das sehe ist die einzig gute die von Shinnosuke Watanabe. Extensions kannst du links in der Leiste installieren, da einfach nach stylelint suchen und halt die von dem Watanbe installieren. Danach solltest du Verbesserungsvorschläge in CSS Dateien angezeigt bekommen. Falls irgendwas nicht klappt, sag nochmal Bescheid!